package.json

```diff
@@ -827,6 +827,15 @@
                             "svg": "resources/walkthrough/createresource.svg",
                             "altText": "Create Azure resources."
                         }
+                    },
+                    {
+                        "id": "ask-azure",
+                        "title": "Ask @azure",
+                        "description": "%walkthrough.askAzure.description%",
+                        "media": {
+                            "image": "resources/walkthrough/askAzure.gif",
```

Comment:
Can you add `resources/walkthrough` to the `.vscodeignore`? IIRC walkthrough images are loaded from the public repository. We could shrink the extension size by a good amount. We should verify that assumption though.

Reply:
I added the line to .vscodeignore. How should I test to make sure it works as expected?

Reply:
Got it. Let me merge the PR and test it via vsix.